src/main/kotlin/com/uber/nanoscope/Nanoscope.kt

```diff
@@ -304,6 +304,48 @@ class Nanoscope {
             println("Opening HTML...")
             Runtime.getRuntime().exec("open $htmlPath")
         }
+
+        private fun displayTraceWithSample(traceFile: File, sampleFile: File, stateFile:File) {
```

Comment:
Is `displayTrace` used anymore? If not, just call this method `displatTrace` and delete the old one.

Reply:
`displayTrace` is used by `nanoscope start` command and by default that won't have sampling enabled. But I think we can just change that one to call this method as well because this method should be robust enough.